elpy.el

```diff
@@ -2050,14 +2064,17 @@ This requires Python 2.7 or later."
 (put 'elpy-test-discover-runner 'elpy-test-runner-p t)
 
 (defun elpy-test-django-runner (top file module test)
-  "Test the project using the Django discover runner.
+  "Test the project using the Django discover runner,
+   or with manage.py if elpy-test-django-with-manage is true.
 
 This requires Django 1.6 or the django-discover-runner package."
   (interactive (elpy-test-at-point))
   (if module
       (apply #'elpy-test-run
              top
-             (append elpy-test-django-runner-command
+             (append (if elpy-test-django-with-manage
+                         elpy-test-django-runner-manage-command
+                       elpy-test-django-runner-command)
                      (list (if test
```

Comment:
So ! I don't clearly see what you suggest. I tried:
```lisp 
(list (concat 
    (expand-file-name (locate-dominating-file (elpy-project-root) "manage.py"))
    ;; returns a directory path, so I must append the file name again.
    ;; This snippet will get more verbose for the need of using the variable `elpy-test-django-runner-manage-command`. 
     "manage.py"))
```
the final command is:

    /home/.../parentdir/projectname/manage.py projectname.myapp.tests-dir.etc

the projectname in the last part is wrong. I suspect it's there because the `top` variable is at `parentdir`.

That's where I am. Anyway, I feel this is a lot of machinery, not necessarily cleaner than keeping the first "if" above, in `elpy-test-at-point` :]